README.rst

```diff
@@ -93,6 +95,8 @@ and we will address it as soon as possible.
 Development Site
 ================
 
+:Version: 2.4.0b1 (development)
```

Comment:
At this point, this information may as well just be part of the prose of this paragraph.